configs/components/rubygem-net-netconf.rb

```diff
@@ -1,17 +1,30 @@
 component "rubygem-net-netconf" do |pkg, settings, platform|
   pkg.version "0.4.3"
-  pkg.md5sum "fa173b0965766a427d8692f6b31c85a4"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/net-netconf-#{pkg.get_version}.gem"
+  if platform.is_huaweios?
+    pkg.url "http://buildsources.delivery.puppetlabs.net/net-netconf-#{pkg.get_version}.tar.gz"
```

Comment:
There are a few gems that let you basically compile native binaries into a gem, so maybe we should look at moving to that method (in the future).